js/main.js

```diff
@@ -1,78 +1,10 @@
-const checkStringLength = (string, maxLength) => string.length < maxLength;
+import './util.js';
+import './data.js';
 
-checkStringLength('Короткий комментарий', 20);
+import {
+  generatePhotos
+} from './data.js';
```

Comment:
Обычно когда импортов мало их располагают в одну строку, в столбик когда 4-5-6 в зависимости от длины
```suggestion
import { generatePhoto } from './data.js';
```